test/acceptance/features/steps/steps.py

```diff
@@ -436,3 +437,25 @@ def validate_persistent_sb(context, sb_name):
         assert True
     else:
         assert False, "Service Binding got updated"
+
+
+@given(u'Secret "{secret_name}" does not contain "{key}"')
+def check_secret_key(context, secret_name, key):
+    openshift = Openshift()
+    json_path = f'{{.data.{key}}}'
+    polling2.poll(lambda: openshift.get_resource_info_by_jsonpath("secrets", secret_name, context.namespace.name,
+                                                                  json_path) == "",
+                  step=5, timeout=120, ignore_exceptions=(binascii.Error,))
+
+
+@given(u'Secret "{secret_name}" is empty')
+@then(u'Secret "{secret_name}" is empty')
+def validate_secret_empty(context, secret_name):
```

Comment:
Instead of two decorators (`given` and `then`), you can use `step` as the decorator.

Reply:
https://github.com/redhat-developer/service-binding-operator/pull/704/commits/13bd37716c3aab868aa8ae9729e83d038405f96a